lib/rules/selector-max-universal/README.md

```diff
@@ -74,3 +74,24 @@ The following patterns are _not_ considered problems:
 /* `*` is inside `:not()`, so it is evaluated separately */
 * > * .foo:not(*) {}
 ```
+
+## Optional secondary options
+
+### `ignoreAfterCombinators: [" ", ">", "*", "~"]`
```

Comment:
```suggestion
### `ignoreAfterCombinators: ["array", "of", "combinators"]`
```

We use this format to describe [options that accept user-defined lists of things to ignore](https://stylelint.io/user-guide/rules/about#secondary) rather than list out the possible values as:

- there be many values, e.g. for an option like `ignoreUnits: []`
- more of the things being targeted may be added to the language, e.g. new combinators, like the shadow-piercing descendant combinator (`>>>`)